drop-in/src/main/java/com/adyen/checkout/dropin/ui/component/BacsDirectDebitDialogFragment.kt

```diff
@@ -78,7 +79,8 @@ internal class BacsDirectDebitDialogFragment : BaseComponentDialogFragment() {
     }
 
     override fun onBackPressed(): Boolean {
-        return if (bacsDirectDebitComponent.delegate.handleBackPress()) {
+        val delegate = bacsDirectDebitComponent.delegate as? BacsDirectDebitDelegate
```

Comment:
You'll be able to call `component.handleBackPress()` directly after [this commit](https://github.com/Adyen/adyen-android/pull/961/commits/1bdbdc8686a14561377eea9caf78b19bf06ee3ef)

Reply:
Oh nice! Didn't see that